test-toolkit/utils/image_utils.py

```diff
@@ -18,50 +18,40 @@
 
 CYAN_COLOR = '\033[36m'
 END_COLOR = '\033[0m'
+DLC_AWS_ID = '763104351884'
 
 
-def build_base_image(framework_name, framework_version, py_version,
-                     processor, base_image_tag, cwd='.'):
-    base_image_uri = get_base_image_uri(framework_name, base_image_tag)
+def build_image(framework_version, dockerfile, image_uri, region, cwd='.'):
+    _check_call('python setup.py sdist')
 
-    dockerfile_location = os.path.join('docker', framework_version, 'base',
-                                       'Dockerfile.{}'.format(processor))
+    if 'dlc' in dockerfile:
+        ecr_login(region, DLC_AWS_ID)
```

Comment:
what's the purpose of this?

Reply:
ECR public images aren't as public as docker hub public images : we still have to do authorization to get one.

Reply:
ah, hadn't quite made the connection that the DLC image is based on the real images 😂